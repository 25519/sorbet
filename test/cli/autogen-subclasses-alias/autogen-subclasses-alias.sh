#!/bin/bash
set -eu

echo "--- autogen-subclasses-alias ---"
main/sorbet --silence-dev-message --stop-after=namer -p autogen-subclasses \
  --autogen-subclasses-parent=Opus::Mixin \
  test/cli/autogen-subclasses-alias/a.rb
