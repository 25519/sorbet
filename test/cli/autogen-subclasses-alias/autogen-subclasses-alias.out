--- autogen-subclasses-alias ---
Opus::Mixin
 Opus::Mixed
 Opus::MixedDescendant
